### DISCUSSION TITLE
需要更新mingw

### DISCUSSION THREAD

**Discussion by ghost** [zhongfly/mpv-build#65] [General]

**Comment by ghost** [comment]
.

**Reply by zhongfly** [reply]
都已经自动全新构建了啊…https://github.com/zhongfly/mpv-winbuild/actions/runs/5395381814

**Reply by zhongfly** [reply]
如果上游更新了工具链，我这里都会自动重新构建的
更新了依赖的话，会单独更新依赖版本

**Reply by zhongfly** [reply]
一会就更新吧

**Comment by zhongfly** [comment]
更新mingw需要重新构建工具链，已开始

https://github.com/zhongfly/mpv-winbuild/actions/runs/5400100596
